cmd/remove.go

```diff
@@ -33,6 +35,10 @@ func RemoveCommand() cli.Command {
 			Name:  "local",
 			Usage: "Deploy Kubernetes cluster locally",
```

Comment:
I know it's not a part of this pr, but should usage say "Remove" instead of "Deploy"?

Reply:
sure will fix this